scripts/lib/mkcloud-dirmaint.sh

```diff
@@ -147,17 +188,52 @@ function dirmaint_do_onhost_deploy_image()
 
 function dirmaint_do_setupadmin()
 {
-    # FIXME
-    echo "NEED TO SETUP ADMIN PROTOTYPE"
+    echo "Setting up admin user"
 
     local admuser=${cloud}adm
     local cloudbr=${cloud}
+    # default directory entry for this user:
+    cat <<EOF > $HOME/${admuser^^}.DIRECT
+USER $admuser cldpaswd 2G 4G G
+  INCLUDE LNXDFLT
+  COMMAND SET SECUSER OPERATOR
+  IPL 100 PARM AUTOCR
+  OPTION LNKNOPAS
+  NICDEF 1000 TYPE QDIO LAN SYSTEM $cloudbr MACID 0${cloudidx}7700
+EOF
 
-    vmcp q $admuser && complain 192 "$admuser is not logged off"
+    # now, lets add this user to the directory:
+    $lxc -c "dirm add $admuser"
+    # the user does not have a disk yet, we might want to add one at virtual
+    # address 100:
+    # FIXME: this is CKD (3390) only, also fixed size of 10015 cylinders
+    # FIXME: for FBA, the type would be 9336, and counting would be 512 byte
+    # blocks.
+    $lxc -c "dirm for $admuser amdisk 0100 3390 autog 10015 CLD9 mr"
```

Comment:
there is a $cloudvg environment variable that the libvirt backend is respecting (it is defaulting to "cloud" in the mkcloud startup code. we could reuse that one here .. maybe add a sanity check that verifies that the poolname set in $cloudvg actually exists.